python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -1,58 +1,105 @@
+The basic datatype used during INT8 inference is signed INT8, with possible values from -128 to 127. However, if `conv2d` or `fc` operator has `relu` or `relu6` activation integrated in it, the output of the operator is known to have non-negative values. In that case we use unsigned INT8 datatype for output tensors, with a wider range for positive values (0 to 255), improving the inference accuracy further.
+
+### Propagation of scales
+
+Some of the operators (e.g. `reshape2`, `transpose2`, `pool2d` with max pooling) transform the data without changing the quantization scale. For this reason we propagate the quantization scale values through these operators without any modifications. We propagate the quantization scales also through the `scale` operator, updating the quantization scale accordingly. This approach lets us minimize the number of `fake_quantize` and `fake_dequantize` operators in the graph, because the information about the scales required for the quantization process to succeed spreads between quantized operators.
+
+### Applying quantization passes
+
+Having gathered all the data needed for quantization we apply the `cpu_quantize_pass` which quantizes the graph, and the `cpu_quantize_squash_pass` which optimizes the INT8 graph.
+
+## 4. Code example
+
+The code snipped shows how the `Qat2Int8MkldnnPass` can be applied to a model graph:
 
 ```python
     import paddle.fluid as fluid
-    from paddle.fluid.contrib.slim.quantization import QatInt8MkldnnPass
     from paddle.fluid.contrib.slim.quantization import Qat2Int8MkldnnPass
     from paddle.fluid.framework import IrGraph
     from paddle.fluid import core	
     
     # Create the IrGraph by Program
     graph = IrGraph(core.Graph(fluid.Program().desc), for_test=False)
     place = fluid.CPUPlace()
-    # Convert the IrGraph to MKL-DNN supported INT8 IrGraph by using
-    # QAT1.0 MKL-DNN
-    # QatInt8MkldnnPass
-    mkldnn_pass = QatInt8MkldnnPass(fluid.global_scope(), place)
-    # Apply QatInt8MkldnnPass to IrGraph
-    mkldnn_pass.apply(graph)
-    # QAT2.0 MKL-DNN
-    # Qat2Int8MkldnnPass, it requires a list of operators to be quantized
+    # Convert the IrGraph to MKL-DNN supported INT8 IrGraph using the
+    # Qat2Int8MkldnnPass. It requires a list of operators to be quantized
```

Comment:
Can you explain how to set the list of operators to be quantized?

Reply:
Done. Added also a note in the *Gathering scales* section.